contrib/devtools/colorize.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+# Copyright (c) 2022 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+'''
+Colorize log output
+Usage: ./bitcoind | contrib/devtools/colorize.py
+'''
+import re
+
+colors = ("cyan", "red", "blue", "green", "magenta")
+color_codes = ("\033[36m", "\033[31m", "\033[34m", "\033[32m", "\033[35m")
+
+def colorize(text):
+    # Colorize groups of brackets
+    for i, match in enumerate(re.finditer(r"\[.*?\]", text)):
+        text = text.replace(match.group(0), color_codes[(i + 1) % len(colors)] + match.group(0) + "\033[0m")
```

Comment:
Since colors is being used only for calculate `len(colors)`, it can be removed.
```suggestion
        text = text.replace(match.group(0), color_codes[(i + 1) % len(color_codes)] + match.group(0) + "\033[0m")
```